criterion-measurement/criterion-measurement.cabal

```diff
@@ -0,0 +1,68 @@
+name:                criterion-measurement
+version:             0.1.0.0
+synopsis:            Criterion measurement functionality and associated types
+description:         Measurement-related functionality extracted from Criterion, with minimal dependencies. The rationale for this is to enable alternative analysis front-ends.
+homepage:            https://github.com/bos/criterion
+license:             BSD3
+license-file:        LICENSE
```

Comment:
There's not a `LICENSE` file alongside `criterion-measurement.cabal`, which is causing build failures on Travis.

Reply:
Added copy of main license